File: CppB/pb2.cpp
//Anushikha Sharma and Cristal Hermosillio
//CSCI208 - Thursday 1pm-3pm
//Prof. Razet
//Lab - CppB
//Problem 2

#include <iostream>
using namespace std;
class Pb2 {
public:
  void print();
  void change(int n);
private:
  string fruit = "Apple: ";
  int value = 7;
};
void Pb2::print(){
  cout << fruit  << value << endl;
}
void Pb2::change(int n){
  value = n;
}


int main(){
  Pb2 obj1 = Pb2(); // notice we did NOT use new here
  Pb2 obj2 = obj1;
  obj1.print();
  obj2.print();
  obj1.change(42); // change a member data to 42
  obj1.print();
  obj2.print();
}

File: CppB/Pb2.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package CppBProblem2;

/**
 *
 * @author Anushikha and Cristal
 */
public class Pb2 {
    public String fruit = "Apple";
    int value = 7;
    public void print() {
      System.out.println("Apple: "+ value);
   }
    public void change(int a) {
        value = a;
   }

public static void main(String[] args) {
    Pb2 obj1 = new Pb2();
    Pb2 obj2 = obj1;
    obj1.print();
    obj2.print();
    obj1.change(42); // change a member data to 42
    obj1.print();
    obj2.print();
}
}
